docs/website/root/mithril/threat-model.md

```diff
@@ -0,0 +1,318 @@
+
+#### Cardano Chain Database
+
+A cardano-node maintains an on-disk database consisting of the chain's history. This database is updated by the node when new blocks are diffused through the network, or minted, and also contains a cache of the ledger state. 
+
+Mithril signer needs needs access to _trusted_ and _up-to-date_ Chain database in order to be able to sign snapshots. 
+
+* **availability**: Yes
+* **integrity**: Yes
+* **Confidentiality**: No (Data is public and replicated) 
+
+#### Cardano Ledger state
+
+Access to an accurate ledger state is needed by Mithril signer to retrieve reliable _Stake distribution_. This access is currently done through a local connection (direct w/ Pallas or indirect with cardano-cli) to a trusted cardano-node
+
+* **availability**: Yes (without SD, signer cannot register keys nor validly use other signers' keys)
+* **integrity**: Yes (same, inaccurate SD will make key registration and signing process invalid)
+* **confidentiality**: No
+
+#### Mithril signing keys
+
+SPOs register their Mithril keys every epoch to be able to sign snapshots. An attacker could impersonate the SPO and sign invalid snapshots if they got hold of those keys.
+Signing keys are currently stored on-disk (?) 
+
+* **confidentiality**: Yes 
+* **integrity**: Yes (invalid key is useless obviously)
+* **availability**: Yes (Signer needs Key at every signing round, unavailability will lead to inability to sign)
+
+#### Mithril signing keys registration
+
+Mithril signer needs to register new key every epoch with aggregator (and ultimately other signers)
+
+* **confidentiality**: Yes (access to a signer's key will allow an attacker to impersonate a signer for the duration of the epoch)
+* **integrity**: Yes (partial? key registration process is transient and limited in time) 
+* **availability**: Yes (need access to aggregator to register key) 
+
+#### Mithril signatures diffusion
+
+Mithril signers produces signatures every time a new message needs to be signed (e.g. when a new immutable file is created in the Chain DB). Those signatures are generated from a random lottery based on the signer's stake and the protocol parameters.
+Preventing Mithril signers from signing decreases the number of signatures and could allow attacker to take control of the produced snapshot
+
+* **confidentiality**: No 
+* **availability**: Yes
+* **integrity**: No (signatures are considered tamper-proof)
+
+#### Mithril protocol parameters 
+
+Protocol parameters are needed to coordinate the production of valid multi-signatures. They are served by the aggregator
+
+* **confidentiality**: No (they actually need to be public)
+* **availability**: Yes (partial?)
+* **integrity**: Yes (tampering protocol parameters can lead a signer to produce invalid signatures)
+
+#### Era configuration files
```

Comment:
This era-related assets should be captured by the system analysis section. Either link to a piece of documentation or explain there how this works and what it is used for.

Reply:
I added a link to the corresponding ADR :+1: